teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
```

Comment:
- [ ]  Please kindly pass `age, name, and parent_permission` arguments in initialize then call it in `super` 🙃 🙃

Reply:
well resolved! ✔️